tests/baselines/reference/privateNamesAndkeyof.types

```diff
@@ -15,6 +26,6 @@ class A {
 >3 : 3
 }
 
-type T = keyof A     // should not include '#foo'
+type T = keyof A     // should not include '#foo*'

```

Comment:
Unfortunately, we changed how we write the type of `keyof A` here, so the only way to validate it now is using assignability.